src/bridge/secrets/data/airbyte/rds.production.yaml

```diff
@@ -0,0 +1,77 @@
+---
+password: ENC[AES256_GCM,data:OQD2ZcC0TraHwt+1s7G9Gh89fCDYSOPZZbkkBcrp9pLqr/5QoFEoPFvpeLlJhq8I,iv:og23WTNucTdU7GM//N60pZfPPX3JhL+NcZTGAjwb1pE=,tag:qeXZSAkmpRy2/1m5JjkxqQ==,type:str]
```

Comment:
Since this is a simple secret that is only valid for the context of Pulumi it's actually better to just use the Stack for encrypting and managing this value. You can use the `--secret` flag to config set to manage the encryption https://www.pulumi.com/docs/reference/cli/pulumi_config_set/